.rubocop.yml

```diff
@@ -1,2 +1,7 @@
 AllCops:
   TargetRubyVersion: 2.3
+
+inherit_gem:
+  rubocop-govuk:
+    - config/default.yml
+    # - config/rails.yml # add this once we're ready to add Rails cops
```

Comment:
For context, the reason we're not adding this now (across all the apps, not just this one) is that `govuk-lint` had Rails cops disabled by default and enabling them during the migration means a change to behaviour and a lot more work to do to fix the linting errors.